freeplane/src/main/java/org/freeplane/view/swing/map/overview/ResizableBorder.java

```diff
@@ -0,0 +1,146 @@
+package org.freeplane.view.swing.map.overview;
+
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Cursor;
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.Insets;
+import java.awt.Point;
+import java.awt.Rectangle;
+import java.awt.event.MouseEvent;
+
+import javax.swing.SwingConstants;
+import javax.swing.border.Border;
+
+import org.freeplane.view.swing.map.MapView;
+
+public class ResizableBorder implements Border, SwingConstants {
```

Comment:
In the best of all possible worlds, I would put ResizableBorder and ResizePanelMouseHandler in their own package because they are not specific to the map overview

Reply:
Will be treated in next commits.